packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -92,10 +93,12 @@ module.exports = function ({ resources, options }) {
   if (path && path.endsWith("/")) {
     path = path.slice(0, -1);
   }
+  const limit = env.limit || (options.configuration && options.configuration.limit);
```

Comment:
shouldn't the default of "1mb" get set here in case both `env.limit` and no `configuration.limit` is set?

Reply:
undefined = 1mb

Reply:
👍